tox.ini

```diff
@@ -11,24 +11,32 @@ deps =
   mypy: mypy~=0.711
 
 setenv =
-  PYTHONPATH={toxinidir}/opentelemetry-api/src/
-  mypy: MYPYPATH={env:PYTHONPATH}
+  mypy: MYPYPATH={toxinidir}/opentelemetry-api/src/
 
 changedir =
-  test: opentelemetry-api/tests
+  test-api: opentelemetry-api/tests
+  test-sdk: opentelemetry-sdk/tests
+
+commands_pre =
+  test-api: pip install -e {toxinidir}/opentelemetry-api
+  test-sdk: pip install -e {toxinidir}/opentelemetry-api
+  test-sdk: pip install -e {toxinidir}/opentelemetry-sdk
 
 commands =
-  py37-mypy: mypy opentelemetry-api/src/opentelemetry/
+  mypy: mypy --namespace-packages opentelemetry-api/src/opentelemetry/
```

Comment:
See https://github.com/python/mypy/issues/1645, mypy fails to find `opentelemetry.api` without this flag.